src/test/java/org/jabref/logic/integrity/EditionCheckerTest.java

```diff
@@ -0,0 +1,23 @@
+package org.jabref.logic.integrity;
+
+import org.jabref.model.database.BibDatabaseContext;
+import org.junit.jupiter.api.Test;
+
+
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+public class EditionCheckerTest {
+     boolean allowIntegerEdition;
```

Comment:
Please no global variable in tests - the variables should be as local as possible.

Reply:
Reduced the scope  to local